packages/components/checkboxes/checkboxes.js

```diff
@@ -0,0 +1,20 @@
+import { toggleConditionalInput } from '../../common';
+
+/*
+* NHS.UK checkbox buttons.
+*
+* Conditionally show content when a checkbox button is checked
+*/
+
+export default () => {
+  // All checkbox inputs inside a conditional form group
+  const checkboxInputs = document.querySelectorAll('.nhsuk-checkboxes--conditional .nhsuk-checkboxes__input');
+
+  // Add event listener to every checkbox button
+  checkboxInputs.forEach((checkboxButton) => {
```

Comment:
Have you tried this on IE11? We might need to add the NodeList.forEach polyfill.
I notice that the details component does some sort of casting to an Array: https://github.com/nhsuk/nhsuk-frontend/blob/master/packages/components/details/details.js#L71

Reply:
I have pulled over the same casting from Details... I will address how it gets used when I pick up current issues with Details component. I think its worth adding the polyfill as it is tiny

Reply:
Maybe worth us investigating the Babel config on here a bit for a balance of size and features?